### DISCUSSION TITLE
createMany giving Typescript error

### DISCUSSION THREAD

**Discussion by rahul3103** [prisma/prisma#17194] [Q&A]
```
async function insertLeaguesData(data: League[]) {
  try {
    await prisma.league.createMany({
      data,
      skipDuplicates: true,
    });
  } catch (e) {
    console.log(
      `${e}`
    );
  }
}
```
Why I am not able to create multiple leagues with `createMany`

```
(property) skipDuplicates: never
Type 'boolean' is not assignable to type 'never'.ts(2322)
```
Model League
```
model League {
  id             String          @id @default(auto()) @map("_id") @fifa_db.ObjectId
  fifaId         Int             @unique
  name           String
  flag           String?
  level          Int?
  nation         Nation          @relation(fields: [nationFifaId], references: [fifaId])
  nationFifaId   Int
  gender         String          @default("male") // number
  LeagueVersions LeagueVersion[]
  TeamVersion    TeamVersion[]

  @@map("leagues")
}
```

**Comment by nurul3101** [comment]
Hey @rahul3103 👋 

Which database are you using?

skipDuplicates is supported only for databases that support `ON CONFLICT DO NOTHING` statement.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
